installer/windows/Product.wxs

```diff
@@ -66,7 +66,10 @@
         <RemoveFile Id="ConfigFile" Name="config.yaml" On="uninstall" />
       </Component>
       <Component Id="dashboardShortcut" Guid="{C48A862E-869D-4FD4-A215-D47092A40BAA}" Win64="yes">
-        <Shortcut Id="DashboardShortcut"  Name="Storage Node Dashboard" Description="Open the dashboard for the storage node" Directory="StorageNodeStartMenuFolder" Target="[System64Folder]cmd.exe" Arguments="/K &quot;&quot;[INSTALLFOLDER]storagenode.exe&quot; dashboard --config-dir &quot;[INSTALLFOLDER]\&quot; --identity-dir &quot;[IDENTITYDIR]\&quot; --color --log.output stdout&quot;">
+        <!-- We use a standard shortcut to 'cmd.exe /C "start http://localhost:14002"' instead of util:InternetShortcut, 
+             so it appears as an App Shortcut with icon in the Windows Start Menu. In contrast, InternetShortcut does
+             not support icons in the Start Menu. -->
+        <Shortcut Id="DashboardShortcut"  Name="Storage Node Dashboard" Description="Open the dashboard for the storage node" Directory="StorageNodeStartMenuFolder" Target="[System64Folder]cmd.exe" Arguments="/C &quot;start http://localhost:14002&quot;">
```

Comment:
this port could be changed via config; it might be nice to have storagenode.exe kick off 'start' and guarantee the right port

Reply:
The installer does not advertise this configuration in the wizard. When the installation completes, this shortcut works properly. The SNO can change the port later in the config.yaml, but the same way they can edit the shortcut too.